crates/nu-protocol/Cargo.toml

```diff
@@ -25,19 +26,25 @@ indexmap = { workspace = true }
 lru = { workspace = true }
 miette = { workspace = true, features = ["fancy-no-backtrace"] }
 num-format = { workspace = true }
+rmp-serde = { workspace = true, optional = true }
 serde = { workspace = true, default-features = false }
 serde_json = { workspace = true, optional = true }
 thiserror = "1.0"
 typetag = "0.2"
 
 [features]
-plugin = ["serde_json"]
+plugin = [
+  "brotli",
+  "rmp-serde",
+  "serde_json",
+]
 
 [dev-dependencies]
 serde_json = { workspace = true }
 strum = "0.26"
 strum_macros = "0.26"
 nu-test-support = { path = "../nu-test-support", version = "0.92.3" }
+pretty_assertions = "1.0"
```

Comment:
I'm wondering if pretty_assertions is necessary? We try to keep dependencies to a minimum. I'm just not sure of the value here.

Reply:
It's just for the tests, we're using it in other places, and it's a very large thing to `assert_eq!` that I'd probably otherwise have to pull apart and compare individual pieces of to be useful